sonic_installer/main.py

```diff
@@ -256,6 +265,7 @@ def umount_next_image_fs(mount_point):
     except SonicRuntimeException as ex:
         click.secho("Warning: SONiC environment variables are not supported for this image: {0}".format(str(ex)),
                     fg="red", err=True)
+        write_msg_cli_syslog("Warning: SONiC environment variables are not supported for this image: {0}".format(str(ex)), "error")
```

Comment:
>write_msg_cli_syslog [](start = 8, length = 20)

Will you remove above line? I guess you need to support `fg`, `err`, etc in your new function